CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/scatter/ArrangeInScatter3dGeneralPlugin.java

```diff
@@ -79,6 +80,13 @@ public void edit(final GraphWriteMethods graph, final PluginInteraction interact
         final String yDimensionName = pp.get(SCATTER_3D_Y_ATTRIBUTE).getStringValue();
         final String zDimensionName = pp.get(SCATTER_3D_Z_ATTRIBUTE).getStringValue();
 
+        if (xDimensionName == null || xDimensionName.equals("") ||
```

Comment:
Please use `StringUtils.isEmpty()`. @antares1470 did a sweeping change to the code base and did this conversion so for consistency and a little bit more readability please use this.

Reply:
Actually use `StringUtils.isBlank()` as this will include the null check as well :)
It will simplify the boolean condition to 
`StringUtils.isBlank(xDimensionName) || StringUtils.isBlank(yDimensionName) || StringUtils.isBlank(zDimensionName)`